### PR TITLE
Better usage of timedelta/time modules

### PR DESCRIPTION
**Pull request by eumiro**
Individual bits to improve the usage and readability of appropriate time/timedelta classes.

### CODE REVIEW

File: py3status/modules/mpris.py [hunk 1]
```diff
@@ -117,14 +117,8 @@
...
-    seconds = int(microseconds / 1000000)
-    time = str(timedelta(seconds=seconds))
-    if time[0] == "0":
-        time = time[2:]
-        if time[0] == "0":
-            time = time[1:]
-
-    return time
+    delta = timedelta(seconds=microseconds // 1_000_000)
+    return str(delta).lstrip("0").lstrip(":").lstrip("0")
```
Comment (lasers) [inline py3status/modules/mpris.py:121]: (Suggestion) `lstrip("0:")`?

Comment (eumiro) [reply py3status/modules/mpris.py:121]: `datetime.timedelta` is definitely missing a good `__format__` support. Both my and your suggestions change the behaviour for timedeltas shorter than 1 minute:

```
microseconds:
    'original'
    'mine'
    'yours'

123:
    '0:00'
    ':00'
    ''

1234567:  
    '0:01'
    ':01'
    '1'

12345678:
    '0:12'
    ':12'
    '12'
```

I'll change it to the original behavior and then we'll wait until `timedelta` can do the job itself.

Comment (eumiro) [reply py3status/modules/mpris.py]: I have changed it and repushed.

**Comment by ultrabug** [comment]
:+1:

**Pull request merged**